code/game/objects/structures/bookcase.dm

```diff
@@ -26,6 +45,10 @@
 		else
```

Comment:
Change `var/newname = stripped_input(usr, "What would you like to title this bookshelf?")` to use `user` instead please

Reply:
done